simulations.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Classes for Static and Dynamic Simulations, and for Simulation Sets
+
+  Typical usage example:
+  sim_one = DynamicSimulation(600, -2, "userwalking.csv")
+  sim_two = StaticSimulation(60, -2, 27.417747, -112.086086)
+  sim_three = StaticSimulation(20, -4, 37.417747, -122.086086)
+  simulation_set = SimulationSet([sim_one, sim_two, sim_three])
+"""
+
+import datetime
+import subprocess
+import time
+
+from tools import kbhit
+
+KEYBOARD = kbhit.KBHit()
+
+
+class Simulation():
+  """An object for a single GPS Simulation.
+
+  Attributes:
+    run_time: simulation duration in seconds
+    gain: signal gain for the broadcast by bladeRF board
+    process: python subprocess created for the simulation
+    process_args: arguments for command to pass into subprocess to run the
+    simulation in bladeGPS
+    start_time: DateTime object for most recent start time
+    end_time: DateTime object for most recent end time
+  """
+
+  def __init__(self, run_time, gain=-2):
+    self.run_time = run_time
+    self.gain = gain
+    self.process = None
+    self.process_args = ["./run_bladerfGPS.sh", "-T", "now", "-d", str(self.run_time)]
```

Comment:
I have both blade RF and bladeGPS make'd inside the project folder

Reply:
Am I missing something, or are they not checked in (https://github.com/googleinterns/geobeam)? Not something that needs to be addressed now, there should probably be some instructions to set it up later on or add them as submodules.

Reply:
No they aren't, I left them out because I figured there were some restrictions on adding them to the repository and that it could just be done as part of setup. There is a document on setup, i can share it with you.